go/oasis-test-runner/oasis/oasis.go

```diff
@@ -806,7 +812,8 @@ func nodeLogPath(dir *env.Dir) string {
 }
 
 func internalSocketPath(dir *env.Dir) string {
-	return filepath.Join(dir.String(), internalSocketFile)
+	p := filepath.Join(dir.String(), internalSocketFile)
```

Comment:
Is it necessary to append `internalSocketFile` since we're just going to use the digest? Why not just use `sha256(dir)`?

Reply:
I thought we could have multiple `internal.sock` addresses for the same client, but I guess that won't be the case in the future.

In addition, I now only use `sha256(dir)[0:4]` to reduce clutter.